docs/tools/cli/byoc.md

```diff
@@ -0,0 +1,127 @@
+---
+title: avn byoc
+---
+
+Set up and manage your [custom clouds](/docs/platform/concepts/byoc) using the Aiven client and `avn byoc` commands.
+
+## Manage a custom cloud
+
+### `avn byoc create`
+
+[Creates a custom cloud](/docs/platform/howto/byoc/create-custom-cloud) in an organization.
+
+| Parameter           | Required | Information                                                                                                           |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where to create a custom clouds                                                         |
+| `--deployment-model`| Yes      | Private or public network architecture model defining how resources are arranged and connected to your cloud provider |
+| `--cloud-provider`  | Yes      | Cloud provider to be used for running a custom cloud                                                                  |
+| `--cloud-region`    | Yes      | Cloud region where to create a custom cloud                                                                           |
+| `--reserved-cidr`   | Yes      | IP address range of the VPC to be created in your cloud account for Aiven services hosted on a custom cloud           |
+| `--display-name`    | Yes      | Name of a custom cloud                                                                                                |
+
+### `avn byoc delete`
+
+Deletes a custom cloud from an organization.
+
+| Parameter           | Required | Information                                                             |
+| ------------------- | -------- | ----------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where to delete a custom cloud            |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud to be deleted                              |
+
+### `avn byoc list`
+
+Returns a list of all the custom clouds in an organization.
+
+| Parameter          | Required | Information                                                              |
+| ------------------ | -------- | ------------------------------------------------------------------------ |
+| `--organization-id`| Yes      | Identifier of an organization where to check for available custom clouds |
```

Comment:
I think "check for available custom clouds" is a bit unclear and IMO not needed. "Identifier of the organization" would be enough of a description in my opionion.